frontend/client_sdk/src/CodaSDK.re

```diff
@@ -51,7 +51,7 @@ type payment = {
 };
 
 type codaSDK;
-[@bs.module "./client_sdk.bc.js"] external codaSDK: codaSDK = "codaSDK";
+[@bs.module "./client_sdk.bc.js"] external codaSDK: codaSDK = "minaSDK";
```

Comment:
reviewer: is this right?

Reply:
yup this works. I will follow up with a name update after this gets merged to convert all references of coda to mina.